core/block/editor/file/file.go

```diff
@@ -68,11 +68,13 @@ type File interface {
 }
 
 type FileSource struct {
-	Path    string
-	Url     string
-	Bytes   []byte
-	Name    string
-	GroupId string
+	Path          string
+	Url           string // nolint:revive
+	Bytes         []byte
+	Name          string
+	GroupID       string
+	WithFileStyle bool
+	FileStyle     model.BlockContentFileStyle
```

Comment:
Seems like FileStyle is a constant equals to model.BlockContentFile_Auto. Does WithFileStyle has to be named WithAutoStyleInFileBlock?